tests/integration/test_integration_app.py

```diff
@@ -1,49 +0,0 @@
-import trello_items as trello
```

Comment:
Did you give up on app integration tests because you couldn't get the mongomock to work?

You would need to make the whole DB connection string an env var (or at least the server name) and the mock server needs to match whatever you put in .env.test